docker-compose.yml

```
@@ -0,0 +1,21 @@
+ services:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)